types/ceddl__ceddl-polyfill/index.d.ts

```diff
@@ -0,0 +1,150 @@
+// Type definitions for @ceddl/ceddl-polyfill 0.9
+// Project: https://www.ceddlbyexample.com/
+// Definitions by: Roland Broekema <https://github.com/broekema41>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 4.4
```

Comment:
```suggestion
```
the 4.4 is the latest, greatest version, why you want this version? (this limits how nice it plays with other typescript)

Reply:
will change this one to 2.2

Reply:
oh, no just leave blank (remove completely)